### PR TITLE
Add `osu! World Cup 2024: Quarterfinals Recap` news post

### PR DESCRIPTION
**Pull request by 0x84f**
yeah

[preview](https://osu2.roanh.dev/home/news/2024-11-20-osu-world-cup-2024-quarterfinals-recap)

### CODE REVIEW

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: "osu! World Cup 2024: Quarterfinals Recap"
+date: 2024-11-20 23:00:00 +0000
+---
+
+First eliminations are in! Read on for the most exciting matchups and to find out which teams are leaving the world cup stage.
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md]: as discussed in the thread these are not the first eliminations

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    }
+</style>
+
+**[Download the pool here! (214 MB)](https://osu.ppy.sh/beatmaps/packs/P266)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2298658144&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+For the first time this year, the pool was expanded quite a bit, with all mods receiving 1 more pick! This means the mappoolers had more chances to isolate specific skill sets, and allow for more specialised players to shine! Alongside the expansions, the pool continues to be populated with throwbacks, the big, black elephant in the room being the first thing to tackle. Yes, that is [Mismagius](https://osu.ppy.sh/users/19048)'s original, iconic map of [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891) in our Free Mod pool. It was picked or banned in over half of the matches this round.
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:35]:
```suggestion
For the first time this year the map pool received expansions, with all mods getting 1 more pick! This means the mappoolers had more chances to isolate specific skill sets, and allow for more specialised players to shine! Alongside the expansions, the pool continues to be populated with throwbacks, with the big, black elephant in the room being the first thing to address. Yes, that is [Mismagius](https://osu.ppy.sh/users/19048)'s original, iconic map of [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891) in our Free Mod pool. Unsurprisingly, it was picked or banned in over half of the matches this round.
```

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+### ::{ flag=PH }:: Philippines vs. ::{ flag=UA }:: Ukraine
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116222613)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2304248553)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cld2spAZQ-M?start=792&end=947" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The best match of the Quarterfinals may have been one you didn't even know about or watch. With three matches all happening at once, ::{ flag=PH }:: Philippines (seed #9) vs. ::{ flag=UA }:: Ukraine (seed #22), drew the short straw of not being streamed on the official Twitch channels. As it happened, this unofficial stream would include the only tiebreaker match of the weekend
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:138]: what unofficial stream? should it be hyperlinked or?
```suggestion
The best match of the Quarterfinals may have been one you didn't even know about or watch. With three matches all happening at once, ::{ flag=PH }:: Philippines (seed #9) vs. ::{ flag=UA }:: Ukraine (seed #22), drew the short straw of not being streamed on the official Twitch channels. As it happened, this unofficial stream would include the only tiebreaker match of the weekend.
```

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:138]:
```suggestion
Both teams had similar strengths yet different weaknesses coming into this match, resulting in a back-and-forth slugfest. The ::{ flag=PH }:: Philippines opened with [DT3](https://osu.ppy.sh/beatmapsets/2072797#osu/4419768) as their first pick, but ::{ flag=UA }:: Ukraine took it as a break point to start the match, consolidating with their own [DT1](https://osu.ppy.sh/beatmapsets/209170#osu/492285) first pick. The ::{ flag=PH }:: Philippines tried switching gears to [NM4](https://osu.ppy.sh/beatmapsets/2278186#osu/4855557), but with three scores above 700,000, ::{ flag=UA }:: Ukraine ended up with 2 break points for a 3–0 lead.
```

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+Both teams had similar strengths yet different weaknesses coming into this match, resulting in a back-and-forth slugfest. The ::{ flag=PH }:: Philippines opened with [DT3](https://osu.ppy.sh/beatmapsets/2072797#osu/4419768) as their first pick, but ::{ flag=UA }:: Ukraine took it as a break point to start the match, consolidating with their own [DT1](https://osu.ppy.sh/beatmapsets/209170#osu/492285) first pick. The ::{ flag=PH }:: Philippines tried switching gears to [NM4](https://osu.ppy.sh/beatmapsets/2278186#osu/4855557), but with three scores above 700,000, ::{ flag=UA }:: Ukraine ended up with 2 break points for a 3-0 lead.
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:140]:
```suggestion
Momentum quickly shifted with ::{ flag=UA }:: Ukraine's next pick, the breakcore-style speed [FM2](https://osu.ppy.sh/beatmapsets/2278184#osu/4855553). The ::{ flag=PH }:: Philippines took this map, won their own next [HD2](https://osu.ppy.sh/beatmapsets/2278079#osu/4855234) pick with a whopping 905,684 coming from ::{ flag=PH }:: [Rammu](https://osu.ppy.sh/users/10652837), and stole ::{ flag=UA }:: Ukraine's HR2 pick afterwards with a carry performance of 839,244 from ::{ flag=PH }:: [zonelouise](https://osu.ppy.sh/users/1492995), tying the match 3–3.
```

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+Momentum quickly shifted with ::{ flag=UA }:: Ukraine's next pick, the breakcore-style speed [FM2](https://osu.ppy.sh/beatmapsets/2278184#osu/4855553). The ::{ flag=PH }:: Philippines took this map, won their own next [HD2](https://osu.ppy.sh/beatmapsets/2278079#osu/4855234) pick with a whopping 905,684 coming from ::{ flag=PH }:: [Rammu](https://osu.ppy.sh/users/10652837), and stole ::{ flag=UA }:: Ukraine's HR2 pick afterwards with a carry performance of 839,244 from ::{ flag=PH }:: [zonelouise](https://osu.ppy.sh/users/1492995), tying the match 3-3.
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:142]:
```suggestion
Another two very close break points followed on [HD3](https://osu.ppy.sh/beatmapsets/1381824#osu/3004925) and [NM2](https://osu.ppy.sh/beatmapsets/2278182#osu/4855537) with a 200,000 score difference on both maps, bringing the match to 4–4. Both teams had saved a good last pick for this moment, with the ::{ flag=PH }:: Philippines picking [NM5](https://osu.ppy.sh/beatmapsets/2020320#osu/4207467), the AR 5 reading pick, and ::{ flag=UA }:: Ukraine picking [HR3](https://osu.ppy.sh/beatmapsets/908811#osu/1896581), the flow alt tech hybrid map. With the teams tied 5–5, the only tiebreaker of the Quarterfinals weekend was secured.
```

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+Another two very close break points followed on [HD3](https://osu.ppy.sh/beatmapsets/1381824#osu/3004925) and [NM2](https://osu.ppy.sh/beatmapsets/2278182#osu/4855537) with very close 200,000 score differences on both maps, bringing the match to 4-4. Both teams had saved a good last pick for this moment, with the ::{ flag=PH }:: Philippines picking [NM5](https://osu.ppy.sh/beatmapsets/2020320#osu/4207467), the AR 5 reading pick, and ::{ flag=UA }:: Ukraine picking [HR3](https://osu.ppy.sh/beatmapsets/908811#osu/1896581), the flow alt tech hybrid map. With the teams tied 5-5, the only tiebreaker of the Quarterfinals weekend was secured.
+
+Both these teams had something to prove going into this tiebreaker — the ::{ flag=PH }:: Philippines and ::{ flag=UA }:: Ukraine both lost at least one key team member from last year, with [enri](https://osu.ppy.sh/users/8640970) and [hyeok2044](https://osu.ppy.sh/users/8472976) leaving the ::{ flag=PH }:: Philippines and [dench](https://osu.ppy.sh/users/10073635) leaving ::{ flag=UA }:: Ukraine. Repeating or going beyond their final placement last year would mean that these teams can hold their own year to year regardless of roster changes.
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:146]: were enri's and hyeok's flags excluded for a reason (flag change) or?

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md (hunk 1 above)
Comment (0x84f) [reply news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md]: ccdf2ecba947e0555f25fc1cef05e9f77e91a4f2 resolves

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "osu! World Cup 2024: Quarterfinals Recap"
+date: 2024-11-20 23:45:00 +0000
+---
+
+Another week has passed and another elimination round is over. Read on for the most exciting matchups, the teams leaving the world cup stage and where the pick'ems crumbled.
```
Comment (0x84f) [inline news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:7]: oxford comma so hivie stops crying
```suggestion
Another week has passed, and another elimination round is over. Read on for the most exciting matchups, the teams leaving the world cup stage and where the pick'ems crumbled.
```

Comment (Walavouchey) [reply news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:7]: this is the oxford comma

```suggestion
Another week has passed and another elimination round is over. Read on for the most exciting matchups, the teams leaving the world cup stage, and where the pick'ems crumbled.
```

**Review by Hiviexd — APPROVED** [review]

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md (hunk 2 above)
Comment (Walavouchey) [reply news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:35]: rest i don't agree with

```suggestion
For the first time this year, the pool was expanded quite a bit, with all mods receiving 1 more pick! This means the mappoolers had more chances to isolate specific skill sets, and allow for more specialised players to shine! Alongside the expansions, the pool continues to be populated with throwbacks, the big, black elephant in the room being the first thing to address. Yes, that is [Mismagius](https://osu.ppy.sh/users/19048)'s original, iconic map of [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891) in our Free Mod pool. It was picked or banned in over half of the matches this round.
```

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md (hunk 3 above)
Comment (Walavouchey) [reply news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:138]: it's linked under the heading

File: news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md (hunk 7 above)
Comment (0x84f) [reply news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md:146]: https://discord.com/channels/188630481301012481/1293209725034303529/1308941695106416690

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**